app/pages/sections/funding-source/manifest.json

```diff
@@ -16,6 +16,7 @@
       }]
     }
   ],
+  "insetAdvice": ["We can only process your order if you’re paying for it in full using GMS funding. You’ll need to create a separate order for any items that are being paid for using a different funding source.", "Once your order is completed, you'll be able to get a copy of your order summary to include in the contract you send to the supplier."],
```

Comment:
a small one but can you put each string on a new line so it's bit easier to read.